scripts/variantstore/tieout/legacy_wdl/JointGenotypingTasks.wdl

```diff
@@ -0,0 +1,1070 @@
+version 1.0
```

Comment:
Completely added this file to make JointGenotyping.wdl validate (it couldn't find the file, which is zipped up in a dependencies zip).